ui.tests/test-module/specs/title/titleV2.authoring.spec.js

```diff
@@ -0,0 +1,106 @@
+const sitesSelectors = require("../../libs/commons/sitesSelectors");
+
+
+describe('Page - Authoring', function () {
+    const dropTitleInContainer = function () {
+        const dataPath = "/content/forms/af/core-components-it/blank/jcr:content/guideContainer/*",
+            responsiveGridDropZoneSelector = sitesSelectors.overlays.overlay.component + "[data-path='" + dataPath + "']";
+        cy.selectLayer("Edit");
+        cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Title", afConstants.components.forms.resourceType.title);
+        cy.get('body').click(0, 0);
+    }
+
+    const dropTitleInSites = function () {
+        const dataPath = "/content/core-components-examples/library/adaptive-form/title/jcr:content/root/responsivegrid/demo/component/guideContainer/*",
+            responsiveGridDropZoneSelector = sitesSelectors.overlays.overlay.component + "[data-path='" + dataPath + "']";
+        cy.selectLayer("Edit");
+        cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Title", afConstants.components.forms.resourceType.title);
+        cy.get('body').click(0, 0);
+    }
+
+    const testTitleEditDialog = function (titleEditPathSelector, titleDrop, isSites) {
+        if (isSites) {
+            dropTitleInSites();
+            cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + titleEditPathSelector);
+            cy.invokeEditableAction("[data-action='CONFIGURE']");
+            cy.get("[name='./fd:htmlelementType']")
+            .should("exist");
+            cy.get('.cq-dialog-cancel').click();
+            cy.deleteComponentByPath(titleDrop);    
+        } else {
+            dropTitleInContainer();
+            cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + titleEditPathSelector);
+            cy.invokeEditableAction("[data-action='CONFIGURE']");
+            cy.get("[name='./fd:htmlelementType']")
+            .should("exist");
+            cy.get('.cq-dialog-cancel').click();
+            cy.get('[data-path^="/content/forms/af/core-components-it/blank/jcr:content/guideContainer/title"]')
+                .invoke('attr', 'data-path')
+                .then(dataPath => {
+                    const id = dataPath.replace('/content/forms/af/core-components-it/blank/jcr:content/guideContainer/title', '');
+                    const _titleDrop = titleDrop + id;
+                    cy.deleteComponentByPath(dataPath);
+                })
+        }
+
+    }
+
+    context('Open Forms Editor', function () {
+        const pagePath = "/content/forms/af/core-components-it/blank",
+            titleEditPath = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/title",
+            titleEditPathSelector = "[data-path^='" + titleEditPath + "']",
+            titleDrop = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/" + afConstants.components.forms.resourceType.title.split("/").pop();
+
+        beforeEach(function () {
```

Comment:
Can we add test case for the design dialog too ? Basically, you can configure policies related to the formats

Reply:
Added!